components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -163,4 +163,19 @@ -(void)testTitleVisibility {
   XCTAssert(self.bottomNavBar.itemViews.lastObject.label.isHidden);
 }
 
+-(void)testViewForItem {
```

Comment:
Please split this into a test found items (at the correct index) and another for an item that is not found.

Reply:
Sounds good.